src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceRegExp.java

```diff
@@ -463,12 +463,12 @@ public void testIssue32_5() {
     }
 
     public void testIssue32_6() {
-        Option aOpt = null; // violation 'Distance between .* declaration and its first usage is 3.'
-        Option bOpt = null; // violation 'Distance between .* declaration and its first usage is 2.'
+        Option aOpt = null; // ok
+        Option bOpt = null; // ok
         Option cOpt = null;
-        isNull(cOpt); // distance = 1
-        isNull(bOpt); // distance = 2
-        isNull(aOpt); // distance = 3
```

Comment:
why it becomes ok ?
because usage is block of similar statements ?

Reply:
Yes, it is natural that the user doesn't want to break the streak of `isNull` just to get rid of the violation. The check is smart enough to understand this.